src/target/target.c

```diff
@@ -1699,8 +1697,9 @@ static int target_call_timer_callbacks_check_time(int checktime)
 
 	keep_alive();
 
-	struct timeval now;
-	gettimeofday(&now, NULL);
+	int64_t now = timeval_ms();
+	if (next_event)
+		*next_event = now + 1000;
```

Comment:
Is this meant as a safe-guard to ensure this processing repeats at least once per second, even if no callback is scheduled?